com.civicrm.hrjobroles/scss/hrjobroles.scss

```diff
@@ -17,3 +17,10 @@
     position: relative;
   }
 }
+
+
+// workaround to make sure that the texarea always go full-width
+// the !important is necessary to override the #bootstrap-theme style
+.editable-textarea {
+  display: block !important;
```

Comment:
Instead of `!important`, if we add this class inside `.job-roles` namespace it would mean the same thing it seems. Then `!important` can be removed.